# K6-Linked Ubiquitylation Marks Toxic RNA–Protein Crosslinks Induced by Formaldehyde

Authors: Aldwin Suryo Rahmanto1*, Christian J. Blum1*, Claudia Scalera1, Jan B. Heidelberger1, Mikhail Mesitov1, Justus F. Gräf1,3, Ivan Mikicic1, Matthias Ostermaier1, Stefanie Ebersberger1, Nils Krapoth1, Nádia Da Silva Fernandes1, Athanasia Mizi4, Yajie Zhu4, Jia-Xuan Chen1, Chunaram Choudhary3, Argyris Papantonis4, Helle D. Ulrich1, Julian König1, Petra Beli1,2,

Affiliations:
1Institute of Molecular Biology (IMB), 55128 Mainz, Germany
2Institute of Developmental Biology and Neurobiology (IDN), Johannes Gutenberg-Universität, 55128 Mainz, Germany
3Department of Proteomics, The Novo Nordisk Foundation Center for Protein Research, Faculty of Health and Medical Sciences, University of Copenhagen, Blegdamsvej 3B, Copenhagen 2200, Denmark
4Institute of Pathology, University Medical Center Göttingen, 37075 Göttingen, Germany

*These authors contributed equally to this work

Correspondence should be addressed to p.beli@imb-mainz.de


Abstract: Reactive aldehydes are produced by normal cellular metabolism or after alcohol consumption, and accumulate in tissues if aldehyde clearance mechanisms are impaired. Their toxicity has been attributed to the damage they cause to genomic DNA and the subsequent inhibition of transcription. However, whether interference with additional cellular processes contributes to aldehyde toxicity has not been investigated. We demonstrate that formaldehyde induces RNA–protein crosslinks (RPCs) that stall the ribosome and inhibit translation. RPCs in the messenger RNA are recognized by the translating ribosomes, marked by atypical K6-linked ubiquitylation catalyzed by the E3 ligase RNF14, and subsequently resolved by the ubiquitin-dependent unfoldase VCP. Our findings uncover an evolutionary conserved aldehyde-induced stress response pathway that protects against RPCs in the cytoplasm and suggest that RPCs contribute to the cellular and tissue toxicity of reactive aldehydes.


Computational Analysis of MS Data
Processed data were analyzed in the R (4.1.1) software environment. The identified diglycine-lysine peptides were filtered for potential contaminants, reverse-identified peptides, and localization probability (>75%). P-values were calculated by a moderated t-test using the limma package. The resulting p-values were corrected for multiple hypothesis testing using the Benjamini–Hochberg FDR method. For downstream analysis, only regulated ubiquitylation sites with an FDR <0.05 were considered significant. GO enrichment analysis for the ubiquitin-modified proteome was done using the clusterProfiler package in R. P-values were evaluated using clusterProfiler and corrected using the Benjamini–Hochberg FDR method. InterPro domain enrichment analysis was performed using EnrichR. The respective terms with the lowest adjusted p-value based on Fisher’s exact testing and correction for multiple comparisons by the Benjamini–Hochberg method are cited here. The identified protein groups were filtered in a similar manner by potential contaminants, reverse-identified peptides, identification only by site, and unique peptides (>1). The statistical analysis described above for the diglycine-lysine peptides was performed. The identified phosphopeptides were filtered for potential contaminants, localization probability (>75%) and reverse reads. Canonical protein isoforms and phosphorylation site positions were assigned using a Python script written in-house. MaxQuant normalized ratios of 4 independent replicates were tested for significant differential regulation using a moderate t-test in the limma package. P-values were adjusted for multiple testing using the FDR method. Kinase activities were estimated using the KSEA algorithm and the R implementation of KSEApp. GO enrichment analysis for the phosphoproteome was done using ViSEAGO package in R, and p-values were assessed using Fisher's exact test. Kinase substrate annotations were obtained from PhosphoSitePlus (PSP) and the NetworKIN database.50 The kinase substrate analysis was performed with a minimum NetworKIN score of 5 and up- and downregulated phosphorylation sites with an FDR <0.05 were included in the prediction. Phosphosite-specific signature analysis was performed using PTMSEA in R. As input data, FDR values generated from the statistical analysis were transformed and multiplied by the sign (+1/–1) of the averaged log2-transformed fold changes. As identifiers for the phosphorylation sites, the flanking sequence (+/− 7 amino acids) was used. PTMsigDB was used as a reference dataset. The minimum overlap between the PTM and PTMsigDB sets was set to 10. Motif visualization of phosphopeptide sequences was done using the ggseqlogo package in R.
